packages/swap/prisma/schema.prisma

```diff
@@ -317,11 +317,13 @@ model BoostPool {
 }
 
 model ChainTracking {
-  id             Int      @id @default(autoincrement())
-  chain          Chain    @unique
-  height         BigInt   @default(0)
-  blockTrackedAt DateTime @default(now())
-  updatedAt      DateTime @updatedAt @db.Timestamptz(3)
+  id                            Int      @id @default(autoincrement())
+  chain                         Chain    @unique
+  height                        BigInt   @default(0)
+  previousHeight                BigInt   @default(0)
+  blockTrackedAtStateChainBlock Int
```

Comment:
i think we can improve this name. What exactly is this ? the statechain block that saw this event?

Reply:
yes exactly, suggestions are really appreciated.
Also for `isTrackingPassedTwoStateChainBlocks` :-/

Reply:
how about `eventBlock` or `eventWitnessedBlock` ?

Reply:
also is there a reason we need previousHeight as a db entry?

Reply:
updated to `eventWitnessedBlock`